client/src/plus/icon-card/index.scss

```diff
@@ -1,4 +1,5 @@
 @use "../../ui/vars" as *;
+@use "~@mdn/minimalist/sass/mixins/animation" as *;
```

Comment:
We should no longer be using `minimalist`. It has been deprecated with the new design and is no longer being maintained.